lib/systems/parse.nix

```diff
@@ -174,14 +176,17 @@ rec {
   types.kernel = enum (attrValues kernels);
 
   kernels = with execFormats; with kernelFamilies; setTypes types.openKernel {
+    # kernel-less targets
+    elf     = { execFormat = elf;     families = { inherit none; }; };
```

Comment:
elf isn't a kernel - it's an abi

Reply:
And barely and ABI, since that normally refers to the machine code itself not the container.